index.html

```diff
@@ -17,12 +17,17 @@
         <nav class="navbar">
             <a href="#" id="logo">My Logo</a>
             <a href="#" class="menu-icon"><img src="images/Union.png" alt="menu bar"></a>
+            <ul class="menu">
+                <li class="menu-items"><a href="#">Portfolio</a></li>
+                <li class="menu-items"><a href="#">About</a></li>
+                <li class="menu-items"><a href="#">Contact</a></li>
+            </ul>
         </nav>
     </header>
     <section id="headline">
```

Comment:
- [ ] The background image on this section appears at the middle of the screen on desktop size, please check this little but important issue.
![image](https://user-images.githubusercontent.com/20563252/121597615-8979af80-ca06-11eb-9e26-b44a57a4d9f5.png)